### PR TITLE
 Milestone 5: Blog App | Controller specs #5 

### PR DESCRIPTION
**Pull request by betses**
### In this milestone I have:

-   Created a Request spec file for all your controllers.
-   Implemented tests for controllers.
-   Checked for linter errors

### CODE REVIEW

File: spec/requests/posts_request_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
```
Comment (SajjadAhmad14) [inline spec/requests/posts_request_spec.rb:2]: Please make sure spec file should include tests that ensures correct view template is rendered
`For example: expect(response).to render_template(:show)`

Please make sure spec file should include tests that ensures the response body includes the correct placeholder text.
`Example : expect(response.body).to include("Welcome to user index page")`

File: spec/requests/users_request_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
```
Comment (SajjadAhmad14) [inline spec/requests/users_request_spec.rb:2]: Please make sure spec file should include tests that ensures correct view template is rendered
`For example: expect(response).to render_template(:show)`

Please make sure spec file should include tests that ensures the response body includes the correct placeholder text.
`Example : expect(response.body).to include("Welcome to user index page")`

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Betsegaw, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by sumairq — APPROVED** [review]
### Hi @betses , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**